### DISCUSSION TITLE
New research suggests that operational and embodied carbon is mathematically incompatible

### DISCUSSION THREAD

**Discussion by JacobValdemar** [Green-Software-Foundation/discussions#56] [Software Carbon Intensity]
Hello everyone,
I recently read the following conference paper which has led me to question the SCI's validity[^1].

I obviously can't redistribute the paper here because of the license/copyright, but I'll try to summarize its argument.

The paper acknowledges that embodied carbon is an important metric that we want to reduce in green software. However, it shows that embodied and operational carbon are calculated using different scales, which will bias reducing a total towards reducing embodied carbon:

> Key Takeaway: Since operational and embodied carbon are measured on different scales, they are incompatible. Metrics that combine embodied and operational carbon will be biased towards reducing embodied carbon due to the carbon multiplier effect.[^1]

The article specifically mentions the SCI being an example of the pitfall "Combining Metrics (§6)" :

> Combining Metrics (§6). Metrics that combine embodied and operational carbon are not meaningful, since, due to doublecounting, embodied and operational carbon are measured on widely different scales and are thus incompatible.[^1]

What do you think about this? Should the SCI be split into two metrics: one for embodied and one for operational carbon - where the aim would be to reduce both metrics?

[^1]: Bashir, Noman, David Irwin, and Prashant Shenoy. ‘On the Promise and Pitfalls of Optimizing Embodied Carbon’. In Proceedings of the 2nd Workshop on Sustainable Computer Systems, 1–6. HotCarbon ’23. New York, NY, USA: Association for Computing Machinery, Aug. 2023. https://doi.org/10.1145/3604930.3605710.

**Comment by mrchrisadams** [comment]
That paper is linked on the Hot Carbon website below - look for the **Track 4: The Life Cycle of Applications (2:25pm - 4:20pm)**

https://hotcarbon.org/2023/program/
https://hotcarbon.org/2023/pdf/a15-bashir.pdf


There is other work done in these fields to refer to, and some of it is even charted visually. Here's one paper I learned about that Daniel Schien told me about:

Chan, C. A., Gygax, A. F., Leckie, C., Wong, E., Nirmalathas, A., & Hinton, K. (2016). Telecommunications energy and greenhouse gas emissions management for future network growth. Applied Energy, 166, 174–185. https://doi.org/10.1016/j.apenergy.2016.01.007

There's also good work done in other  field to explore these trade-offs.

This chart shows the embodied energy with different refresh cycles along with energy usage over different refresh cycles:

<img width="803" alt="image001" src="https://github.com/Green-Software-Foundation/discussions/assets/17906/d1aa2a7f-063e-4abc-8a87-8d77c1aa0ef9">


That paper also has a chart showing them together, to help understand the likely "sweet spots" that yield energy efficiency savings, without coming up with massive embodied carbon emissions - this chart implies that for the most part, 5 years is the optimum point where carbon emissions savings from efficient kit usage are decent, and are not offset by a refresh cycle that pushes up your embodied carbon:

<img width="1125" alt="image" src="https://github.com/Green-Software-Foundation/discussions/assets/17906/867fbc11-6a17-44b3-b66b-9d324ee41cbc">

Facebook also have their own models they open sourced explioring these trade-offs in accounting for the emissions for their own infrastructure:
https://github.com/facebookresearch/CarbonExplorer/

And here's a nice video from one of the authors explaining that Carbon Explorer work:
https://www.youtube.com/watch?v=gpeZZHgZPkQ

**Comment by navveenb** [comment]
@wg discussions -  From a scoping point, SCI is used more for decision-making and not reporting/disclosures. The current view from the workgroup is splitting SCI into two metrics is not required.

**Reply by JacobValdemar** [reply]
Is the Working Group not afraid that the current model is biased towards reducing embodied carbon and therefore deincentivize reducing operational carbon, or is it an overarching priority to keep the SCI as a single metric? I am asking because I definitely understand if it's a priority to keep it simple, but one should be aware of the tradeoff and the possible bias.

**Reply by NAMRATA-WOKE** [reply]
Tagging @navveenb so he catches your response!
